lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -82,9 +82,11 @@ class OptimizedImages extends Gatherer {
       const isSameOrigin = URL.hostsMatch(pageUrl, record._url);
```

Comment:
> if you're doing http -> https we'll complain elsewhere anyway

this is a fair point, but I think it'll just be a matter of time until we get an issue filed on this :) Testing cross origin seems like the right thing to do to me in this case, but also agree the inconsistency between this and other gatherers/audits doing host comparison is less than ideal